virt/virt-2-4-release-notes.adoc

```diff
@@ -37,6 +37,7 @@ include::modules/virt-what-you-can-do-with-virt.adoc[leveloffset=+2]
 //Placeholder for new content.
 
 //CNV-5274 - Customize accessMode and volumeMode when creating VM disks.
+* You can now configure the *Volume Mode* and *Access Mode* for a virtual disk when you xref:../virt/virtual_machines/virt-edit-vms.adoc#virt-vm-add-disk_virt-edit-vms[add a disk to a virtual machine] in the web console. This is also possible when adding a disk to a xref:../virt/virtual_machines/virt-create-vms.adoc#virt-creating-vm-wizard-web_virt-create-vms[new virtual machine using the wizard].
```

Comment:
One suggestion: split this into two bullet points instead of "This is also possible when"

But if that's too awkward, I think this is fine as-is.